### PR TITLE
[19] Style toolbar

### LINKED ISSUE #19: Style toolbar
**As a** user
**I want to** style the toolbar
**So that** the app is more appealing

### PR DESCRIPTION
**Pull request by marcosorive**
Closes #19

### CODE REVIEW

**Comment by marcosorive** [comment]
Toolbar looks like this: imagen

File: src/frontend/app/toolbar/import-file/import-file.component.scss [hunk 1]
```diff
@@ -1,12 +1,18 @@
...
-    display: flex;
-    flex-direction: column;
-    align-items: center;
+.import-file__input{
+    display:none;
 }
 
-label, input {
-    flex: 1 1 auto;
+.import-file__label-container{
```
Comment (jrposada) [inline src/frontend/app/toolbar/import-file/import-file.component.scss]: For this button could you remove the label text and use an icon instead? I would suggest the upload_file from https://material.io/resources/icons/?icon=import_export&style=outline

File: src/frontend/app/toolbar/import-file/import-file.component.html [hunk 2]
```diff
@@ -2,5 +2,11 @@
        id="file"
        [accept]="supportedFileTypes"
        (change)="handleFileInput($event.target?.files)"
-       [formControl]="inputFile">
-<label for="file">Import file</label>
\ No newline at end of file
+       [formControl]="inputFile"
+       class="import-file__input">
+<div class="import-file__label-container">
```
Comment (jrposada) [inline src/frontend/app/toolbar/import-file/import-file.component.html]: Could we use a mat-button instead of the div? Or at least the ripple effect.

Comment (marcosorive) [reply src/frontend/app/toolbar/import-file/import-file.component.html]: Done!

File: src/frontend/app/toolbar/import-file/import-file.component.scss (hunk 1 above)
Comment (marcosorive) [reply src/frontend/app/toolbar/import-file/import-file.component.scss]: Done!

**Comment by marcosorive** [comment]
Toolbar now looks like this: 
imagen

And the buttons have the rippel effect from the material library.

**Review by jrposada — APPROVED** [review]

**Pull request merged**